react-native-background-downloader.podspec

```diff
@@ -14,4 +14,5 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency 'React-Core'
+  s.dependency 'SSZipArchive', '2.2.3'
```

Comment:
I tried updating this to 2.4.3 but it still did not work and the higher versions are limited on what os they run